apps/api/src/app/auth/auth.controller.ts

```diff
@@ -91,7 +91,7 @@ export class AuthController {
      * Make sure we only allow localhost redirects for CLI use and our own success route
      * https://github.com/novuhq/novu/security/code-scanning/3
      */
-    if (redirectUrl && redirectUrl.startsWith('http://localhost:')) {
+    if (redirectUrl && redirectUrl.startsWith('http://localhost:') && !redirectUrl.includes('@')) {
```

Comment:
Are we missing the check for the question mark? Based on the PR title.

Reply:
Me and Dima discussed and said it was not needed after I had created the pr :)

Reply:
No, it was not needed in the end :) I see David updated the title